gh-pages/assets/javascript/index.js

```diff
@@ -0,0 +1,80 @@
+import utils from "./utils.js"
+import { FrenchMap } from "./map.js"
+
+/**
+ * Populate the table containing prices averages
+ * @param {Map<string, number>} averages Map from fuel_type to price
+ * @param {Map<string, string>} fuelNames Map from fuel_type to fuel_name
+ */
+function populateAveragesTable(averages, fuelNames) {
+    let parsedAverages = utils.mapPricesToNames(averages, fuelNames)
+    parsedAverages = parsedAverages.sort((obj1, obj2) => obj2.price - obj1.price) // by descending price
+    utils.populateTable(
+        "table#averages",
+        parsedAverages.map(value => [value.name, value.price.toFixed(2) + " €"])
+    )
+}
+
+/**
+ * Build values usable as input for the Map Visualization
+ * @param {object} averages price averages by department and fuel type
```

Comment:
An example of the object form ?

Reply:
Done https://github.com/Doreapp/prix-carburants/pull/36/commits/28268a09deb1f94c6a3960ed1f2f7cac70dc05a1